libnopegl/src/text.c

```diff
@@ -574,7 +586,12 @@ int ngli_text_set_time(struct text *s, double t)
 
             if ((ret = set_transform( s->data_ptrs.transform  + c * 4 * 4, effect_opts->transform_chain,                       target_t)) < 0 ||
                 (ret = set_vec3_value(s->data_ptrs.color      + c * 4,     effect_opts->color_node,      effect_opts->color,   target_t)) < 0 ||
-                (ret = set_f32_value( s->data_ptrs.color      + c * 4 + 3, effect_opts->opacity_node,    effect_opts->opacity, target_t)) < 0)
+                (ret = set_f32_value( s->data_ptrs.color      + c * 4 + 3, effect_opts->opacity_node,    effect_opts->opacity, target_t)) < 0 ||
+                (ret = set_vec3_value(s->data_ptrs.outline    + c * 4,     effect_opts->outline_color_node, effect_opts->outline_color, target_t)) < 0 ||
+                (ret = set_f32_value( s->data_ptrs.outline    + c * 4 + 4, effect_opts->outline_node,       effect_opts->outline,       target_t)) < 0 ||
```

Comment:
Isn't it c * 4 +3 ?

Reply:
Oops yeah. Funny that it wasn't spotted by the CI...